js/data.js

```diff
@@ -6,3 +6,16 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+var previousEntriesJson = localStorage.getItem('javascript-local-storage');
```

Comment:
You should name this something like `code-journal-data`, not `javascript-local-storage`. Try to make the name describe the purpose of the data.